osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -151,23 +151,27 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
         }
 
         private ScheduledDelegate pushDebounce;
-        private VisualSettings visualSettings;
+        protected VisualSettings VisualSettings;
 
         private bool readyForPush => player.LoadState == LoadState.Ready && IsHovered && GetContainingInputManager()?.DraggedDrawable == null;
 
         protected override bool OnHover(HoverEvent e)
         {
             // restore our screen defaults
             InitializeBackgroundElements();
+            if (this.IsCurrentScreen() && (Background?.IsLoaded ?? false))
```

Comment:
is it actually possible for background to be null here? possibly cause for concern.